hello/templates/hello/playlist_view.html

```diff
@@ -30,7 +44,7 @@ <h4>Star Rating:</h4>
             </div>
         </div>
         <div class="playlist-fillin">
-            Your playlist has been made
+            Now you can go to Spotify and it'll be the first playlist in your library
```

Comment:
could this instead be a little less wordy? "Your playlist has been made" is probably good enough.